### DISCUSSION TITLE
Special unicode characters

### DISCUSSION THREAD

**Discussion by donno2048** [pyinstaller/pyinstaller#5489] [Help] [answered]
I'm writing a program in Python which uses some special Unicode characters.

In Windows, when opened from the command line the special characters are printed as `\u2370` ("⍰").

When opened from Windows terminal I get the excepted interpretation.

The question is: can I pack my Python file in such a way that when opened it will show special Unicode characters?

**Comment by bwoodsend** [comment]
Windows cmd doesn't work with unicode unless you tinker with its settings. Have you tried just running your code with Python in the cmd prompt?
```shell
python my_code.py
```
I really doubt this is PyInstaller specific.

**Reply by donno2048** [reply]
I know that, I just want to know if there is a way to make an executable with Unicode supporting command line out of the box...

**Reply by Legorooj** [reply]
You may be able to use runtime hooks for this.

**Reply by donno2048** [reply]
I have no idea how to do that...

**Reply by htgoebel** [reply]
Please read the manual

**Reply by donno2048** [reply]
How would runtime hooks help? I can't directly pack with fonts or a terminal environment...

**Comment by donno2048** [comment, marked as answer]
CMD doesn't support Unicode by default dumb past-me

**Comment by donno2048** [comment]
CMD doesn't support Unicode by default dumb past-me
